extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/binder/vertx/HttpRequestMetric.java

```diff
@@ -81,4 +85,11 @@ public String toString() {
         return "HttpRequestMetric [initialPath=" + initialPath + ", currentRoutePath=" + currentRoutePath
                 + ", templatePath=" + templatePath + ", request=" + request + "]";
     }
+
+    private boolean isCORSPreflightRequest() {
```

Comment:
@ebullient Hi Erin - if Quarkus HTTP CORS is enabled then the CORS filter will not let the request flow further:

 https://github.com/quarkusio/quarkus/blob/main/extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/cors/CORSFilter.java#L125

Is this `HttpRequestMetric` code running before CORSFilter ?

thanks

Reply:
it is caught @ vertx layer.

go ahead and run the test. ;)

Reply:
@ebullient this is very cool